networking/ingress/configuring-ingress-operator.adoc

```diff
@@ -0,0 +1,35 @@
+[id="configuring-ingress"]
+= Ingress Operator in {product-title}
+include::modules/common-attributes.adoc[]
+:context: configuring-ingress
+
+toc::[]
+
+The Ingress Operator is an OpenShift component that enables external access to
+cluster services by configuring Ingress Controllers. You can use the Ingress
+Operator to route traffic by specifying OpenShift Route and Kubernetes Ingress
+resources.
```

Comment:
Should that be ```OpenShift `Route` and Kubernetes `Ingress` resources``` since it is referring to API resource kinds?

Reply:
Yep - thanks!